openmrs/apps/clinical/app.json

```diff
@@ -36,34 +36,65 @@
             ]
         },
     	"diagnosisStatus": "Inactive",
-            "program": {
-                "patientInformation": {
-                    "ageLimit": 100
-                },
-                "ID_Number": {
-                    "required" : true
-                },
-		        "PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES": {
-                    "excludeFrom": ["TB_PROGRAM_KEY"]
-                },
-                "PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE": {
-                    "excludeFrom": ["TB_PROGRAM_KEY"]
-                }
-            },
-            "patientContext": {
-        		"programAttributes": [
-            		"Id_Number",
-            		"Doctor",
-            		"Stage"
-        		],
-        		"personAttributes": [
-            		"class",
-            		"caste"
-        		],
-        		"additionalPatientIdentifiers": [
-           			"National ID"
+        "program": {
+            "patientInformation": {
+                "ageLimit": 100
+            },
+            "PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES": {
+                "excludeFrom": ["TB_PROGRAM_KEY", "HIV_DEFAULTERS_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE": {
+                "excludeFrom": ["TB_PROGRAM_KEY", "HIV_DEFAULTERS_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_DURATION_ON_ART": {
+                "excludeFrom": ["TB_PROGRAM_KEY", "HIV_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_PATIENT_STAGE": {
+                "excludeFrom": ["TB_PROGRAM_KEY", "HIV_DEFAULTERS_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_PATIENT_DOCTOR_IN_CHARGE": {
+                "excludeFrom": ["HIV_DEFAULTERS_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_ART_NUMBER": {
+                "excludeFrom": ["TB_PROGRAM_KEY", "HIV_DEFAULTERS_PROGRAM_KEY"]
+            },
+            "PROGRAM_MANAGEMENT_REASON_FOR_DEFAULTING": {
```

Comment:
@lauravignoli this field needs to be multi-select (I'm not sure if this is possible to do in the configuration)

Reply:
I haven't found a way to do it. We might have to do yes/no for each reason. Would like to check with Lisa when she is back. I will add a comment on the ticket.